src/lib/config-schema.json

```diff
@@ -1,13 +1,15 @@
 {
   "$id": "https://github.com/massfords/asl-puml#",
   "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Config",
   "type": "object",
   "$defs": {
     "color": {
       "type": "string",
       "pattern": "^#([a-zA-Z]+)|[\\da-fA-F]{2,6}$"
     },
     "compensation": {
+      "title": "Compensation Config",
```

Comment:
adding `title` field to the schemas since this is what's used to create the type name

using the same names for backwards compatibility